### PR TITLE
add example url

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3579

### CODE REVIEW

File: SharePoint/SharePointOnline/remove-search-results.md [hunk 1]
```diff
@@ -36,6 +36,6 @@ As a Global Administrator or SharePoint Administrator in Microsoft 365, you can
    
 3. On the **search administration** page, select **Remove Search Results**.
     
-4. On the **Remove Search Results** page, in the **URLs to remove** box, enter the URLs that you want to remove from the search results. Enter one URL on each line. 
+4. On the **Remove Search Results** page, in the **URLs to remove** box, enter the URLs that you want to remove from the search results. For example, https://contoso.sharepoint.com/sites/site1, URLs cannot contain a wildcard (\*) character. Enter one URL on each line.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/remove-search-results.md:39]:
```suggestion
4. On the **Remove Search Results** page, in the **URLs to remove** box, enter the URLs that you want to remove from the search results, for example, https://contoso.sharepoint.com/sites/site1. URLs cannot contain a wildcard (\*) character. Enter one URL on each line. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@tiburd can you help me here? thanks

**Pull request merged**